modules/Core/src/main/java/org/terasology/core/world/generator/facetProviders/PerlinHillsAndMountainsProvider.java

```diff
@@ -69,8 +69,8 @@ public void process(GeneratingRegion region) {
             float tempHumid = temp * humidityData.get(pos);
             Vector2f distanceToMountainBiome = new Vector2f(temp - 0.25f, tempHumid - 0.35f);
             float mIntens = TeraMath.clamp(1.0f - distanceToMountainBiome.length() * 3.0f);
-            float densityMountains = Math.max(mountainData[i], 0) * mIntens * configuration.mountainAmplitude;
-            float densityHills = Math.max(hillData[i] - 0.1f, 0) * (1.0f - mIntens) * configuration.hillAmplitude;
+            float densityMountains = Math.max(mountainData[i] * 2.12f, 0) * mIntens * configuration.mountainAmplitude;
```

Comment:
2.11f was not good enough?  had to one up the surface provider?  

This is to make it more pronounced?  Should that be delegated to mIntens variable instead?